public/static/js/mci_module.js

```diff
@@ -292,7 +292,7 @@ var mciModule = angular.module('MCI', [
   return function(task) {
     if (task.status == 'started') {
       return 'started';
-    } else if (task.status == 'undispatched' && task.activated) {
+    } else if (task.status == 'undispatched' && task.activated || task.display_only && task.task_waiting) {
```

Comment:
Are you trying to say  `(task.status == 'undispatched' && task.activated) || task.display_only && task.task_waiting)`
or `task.status == 'undispatched' && (task.activated || task.display_only && task.task_waiting)`? Please clarify with parens.

Reply:
done